docker-compose.yml

```
@@ -5,9 +5,9 @@ services:
     container_name: clickhouse
     restart: unless-stopped
     environment:
-      - CLICKHOUSE_USER=${DB_USER}
-      - CLICKHOUSE_PASSWORD=${DB_PASSWORD}
-      - CLICKHOUSE_DB=${DB_NAME}
+      - CLICKHOUSE_USER=default
```

```suggestion
      - CLICKHOUSE_USER=${DB_USER:-default}
```
and for the following vars.
some users can use external clickhouse and the changes shouldn't break their config